### DISCUSSION TITLE
Tell us more about your experience using ORCA!

### DISCUSSION THREAD

**Discussion by karashah1** [cammurray/orca#199] [General]
![image](https://user-images.githubusercontent.com/67892508/155944552-4a33dded-f60c-4f9d-baf8-a6d06c932492.png)

While it's great to hear that you enjoyed using our product, it is also humbling to hear that customers didn't like our product. In either case, we understand your perspective matters a lot! Please share your thoughts on how we can improve your ORCA experience.

**Tell us about:**
- Why do you use ORCA?
- What do you like or not like about ORCA?
- What do you want to see more of or less of?
- What role (admin, leadership, engineer, etc) do you belong to & which other stakeholders in your or use ORCA?
- Any other feedback that you want us to work on.

**Interested in discussing more?**
Leave your email address and we will reach out to you or drop us an email at [cxe-help@microsoft.com](mailto:cxe-help@microsoft.com).

**Comment by SigiJ** [comment]
Hi!

I am using your script quite often, to investigate an infrastructure! Great Work.

Here is some recommendation of issues for v2:
•	Please combine Safe Attachments Policy Rules and Safe Links Policy Rules to one list item, same as you did User Impersonation Action. This would help when summarizing the changes for the customer 😉
•	Default Outbound spam filter policy is incorrectly named - and also NOT system managed.
•	Consider non existing DNS MX domains, they currently show as "Standard/Green" - which is not the case 
•	DKIM, non exisiting DNS MX domains, should show: "No Configuration", they say "Standard/Green" 

If you ask me, I would like to see more about "when" you should follow the rules, and what type of impact that setting has: e.g., mail flow, end user information, admin informational, etc.

**Comment by thunderbirdK** [comment]
Used the script for the fist time and I got the following error

![image](https://github.com/cammurray/orca/assets/12069651/23e50d59-021b-45ba-93d5-4169c8d1d993)

I am using PowerShell 7

**Reply by cammurray** [reply]
Hi @thunderbirdK,

For issues please log in the issues section so we can track them properly, but for this one, it is likely that you either don't have permissions (so that command is being filtered out), or the connection to the security and compliance powershell failed/didnt work for a particular reason.

There is an issue for this already logged by someone else: #263 

Feel free to comment there, have it re-opened, and we can look in to this further. We can also put some logic in place so that if we can't find the command we give some more specific guidance.

**Comment by cammurray** [comment]
Locked this conversation, for issues, please log in the "Issues" table.
